perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -131,6 +131,15 @@
 SECONDS_UNIT = 'seconds'
 MS_UNIT = 'milliseconds'
 
+MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
```

Comment:
Would you like me to move just those necessary variables or all of the variable declarations above flags?
 ie
-----------------
MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
MYSQL_SVC_OLTP_TABLE_SIZE = 'mysql_svc_oltp_table_size'
MYSQL_SVC_DB_INSTANCE_CORES = 'mysql_svc_db_instance_cores'
SYSBENCH_WARM_UP_SECONDS = 'sysbench_warm_up_seconds'
SYSBENCH_RUN_SECONDS = 'sysbench_run_seconds'
SYSBENCH_THREAD_COUNT = 'sysbench_thread_count'
SYSBENCH_LATENCY_PERCENTILE = 'sysbench_latency_percentile'
SYSBENCH_REPORT_INTERVAL = 'sysbench_report_interval'

FLAGS...
BENCHMARK_NAME = 'mysql_service'
BENCHMARK_CONFIG = """
mysql_service:

-------------------
OR 
-------------------
MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
MYSQL_SVC_OLTP_TABLE_SIZE = 'mysql_svc_oltp_table_size'
MYSQL_SVC_DB_INSTANCE_CORES = 'mysql_svc_db_instance_cores'
SYSBENCH_WARM_UP_SECONDS = 'sysbench_warm_up_seconds'
SYSBENCH_RUN_SECONDS = 'sysbench_run_seconds'
SYSBENCH_THREAD_COUNT = 'sysbench_thread_count'
SYSBENCH_LATENCY_PERCENTILE = 'sysbench_latency_percentile'
SYSBENCH_REPORT_INTERVAL = 'sysbench_report_interval'

BENCHMARK_NAME = 'mysql_service'
BENCHMARK_CONFIG = """
mysql_service:
...

FLAGS...